jobseeker-backend/.vscode/settings.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
Don't push your .vscode/*  files to GitHub

Reply:
Thanks